### ISSUE TITLE
Blocker: VM Portal crashes at login time when nic.mac is fetched as null instead of undefined, might occur for other endpoints as well

### ISSUE THREAD

**Issue by sgratch** [oVirt/ovirt-web-ui#1544]
The following exception appears at login time:

![image](https://user-images.githubusercontent.com/18169498/145390189-d4510aa2-ceeb-4cfb-ba17-bfc7438972ea.png)

We need to handle a case when nic.mac is set to null by backend.
I assume this regression is due to ovirt-engine fix: https://gerrit.ovirt.org/117685

**Comment by rszwajko** [comment]
> I assume this regression is due to ovirt-engine fix: https://gerrit.ovirt.org/117685
>
The REST API change seems to be related with patch https://gerrit.ovirt.org/c/ovirt-engine/+/113872

**Comment by sjd78** [comment]
Shouldn't this work (untested):
```js
const { mac: { address: nicMacAddress = '' } = {} } = nic ?? {}
```

**Comment by sgratch** [comment]
The original crash issue was solved and merged by PR#1545 and a [bug](https://bugzilla.redhat.com/show_bug.cgi?id=2032917) was opened on infra for handling all other potential cases.
Therefore I'm closing this issue.
